extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -219,6 +219,11 @@ require([
 		$('.RecentActivityModule').on('mousedown touchstart', 'a', function (event) {
 			handleClick(event, 'community-page-recent-activity-module');
 		});
+
+		// Track clicks in the Insights modules
+		$('.community-page-insights-module').on('mousedown touchstart', 'a', function (event) {
+			handleClick(event, event.delegateTarget.getAttribute('data-tracking'));
```

Comment:
In handleClick we get the 'data-tracking' attribute like so:

   var data = $(event.currentTarget).data('tracking');

One of the places should probably be changed to make it consistent.